modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+// TBD - Where does this release note belong?
+
+* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR) YAML. The Operator adds these values to all deployments it creates.
++
+To configure node selection for the Operator's controller and webhook deployment, you edit the `Subscription` CR after installing the Operator.
+The `Subscription` CR has the following fields in its specification:
+- `config.nodeSelector`
+- `config.toleration`
++
+To deploy on an infra node, you update the `TektonConfig` and `Subscription` CRs with the infra label as `nodeSelector` and `toleration` so that these labels are applied to the infra node. (link:https://issues.redhat.com/browse/SRVKP-1544[SRVKP-1544])
+
+[id="deprecated-features-1-6_{context}"]
+== Deprecated features
+
+=== CLI 0.21.0 release
+
+* In CLI 0.21.0, support for all `v1alpha1` resources for `clustertask`, `task`, `taskrun`, `pipeline`, and `pipelinerun` commands are deprecated. These resources will be removed after two releases.
+
+=== Tekton Triggers v0.16.0
+
+* In Tekton Triggers v0.16.0, the redundant status tag is removed from the `EventListener` metrics. (link:https://github.com/tektoncd/triggers/pull/1166[#1166])
++
+[IMPORTANT]
+====
+Breaking change: The Status label has been removed from the `eventlistener_http_duration_seconds_*` metric.
+Remove queryies that are based on the status label.
+====
+
+=== Tekton Triggers v0.15.0
+
+* With the current release, the `EventListener` sends a response before Triggers finishes processing. (link:https://github.com/tektoncd/triggers/pull/1132[#1132])
++
+[IMPORTANT]
+====
+Breaking change: With this change, the `EventListener` stops responding with a `201 Created` status code when it creates resources. Instead, it responds with a `202 Accepted` response code.
+====
+
+* The current release removes the `podTemplate` field from the `EventListener`. (link:https://github.com/tektoncd/triggers/pull/1118[#1118])
++
+[IMPORTANT]
+====
+Breaking change: The `podTemplate` field, which was deprecated as part of link:https://github.com/tektoncd/triggers/pull/1100[#1100], has been removed.
+====
+
+* The current release removes the deprecated `replicas` field from the `EventListener` specification. (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
+[id="removing-home-workingdir-values-1-6_{context}"]
+=== Removing the HOME="/tekton/home" and workingDir="/workspace" values from Steps
+// TBD - Where does this release note belong?
+
+In {pipelines-title} 1.6, the values of `HOME="/tekton/home"` and `workingDir="/workspace"` are removed from the `Steps` specification.
+
+Instead, {pipelines-title} sets `HOME` and `workingDir` to the values defined by your Steps containers. You can override these values in your `Steps` YAML file.
```

Comment:
consistently format Steps / `Steps` - maybe steps / `Steps` specification as per other objects? Is `Steps` a kind of CR?

Reply:
Updated. Thanks.